deploy/operator/utils.sh

```diff
@@ -26,41 +27,80 @@ function installed_remote_agents() {
 export -f installed_remote_agents
 
 function wait_for_operator() {
-    subscription="$1"
-    namespace="${2:-}"
-    echo "Waiting for operator ${subscription} to get installed on namespace ${namespace}..."
-
-    for _ in $(seq 1 60); do
-        csv=$(oc -n "${namespace}" get subscription "${subscription}" -o jsonpath='{.status.installedCSV}' || true)
-        if [[ -n "${csv}" ]]; then
-            if [[ "$(oc -n "${namespace}" get csv "${csv}" -o jsonpath='{.status.phase}')" == "Succeeded" ]]; then
-                echo "ClusterServiceVersion (${csv}) is ready"
-                return 0
-            fi
-        fi
-
-        sleep 10
-    done
+   subscription="$1"
+   namespace="${2:-}"
+   set -x
```

Comment:
Not sure enabling tracing in inside a utility func is a good idea, maybe let the outer scope script decide whether to enable it or not to avoid unexpected behaviour